packages/ComponentLibrary/src/components/Table/sections/CenterSection.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, useTheme } from '@mui/material';
+import IconButton from '../../IconButton';
+import Excel from '../../../assets/icons/ilustration/excel.svg';
+import Copilot from '../../../assets/icons/sparks.svg';
+import Print from '../../../assets/icons/color-picker.svg';
+import Trash from '../../../assets/icons/trash-2.svg';
+import Printer from '../../../assets/icons/printer.svg';
+import Copy from '../../../assets/icons/copy.svg';
+import Mail from '../../../assets/icons/mail.svg';
+import LinkIcon from '../../../assets/icons/link.svg';
+
+const CenterSection = () => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        width: '100%',
+        background: `var(--Neutral-0, ${theme.palette.baselineColor.transparentNeutral[5]})`,
+        borderRadius: '10rem',
+        padding: '.25rem',
+        gap: '0.25rem',
+      }}>
+      <IconButton tooltip="Print" width={16} height={16}>
```

Comment:
Create an array to map them and not repeat so much of the same code

Reply:
Agree, remember to develop component in a generic way.
It have to receive an array of icons instead

Reply:
@sjcuello @tomyrodeghiero This is fixed in the Feature EPL-1723, since all this files got replaced and improved the structure there